static/js/navbar_alerts.js

```diff
@@ -169,8 +176,8 @@ export function initialize() {
     });
 }
 
-export function open($process) {
-    $("[data-process]").hide();
-    $process.show();
+export function open(render_alert, args = {}) {
+    const rendered_navbar_alert_html = render_alert(args);
+    $("#navbar_alerts_wrapper").html(rendered_navbar_alert_html);
```

Comment:
The API here is a bit awkward; why not do the rendering in the caller and just pass the resulting HTML object into this function?  I think that'd be a bit cleaner.

It's worth adding a comment here to make clear that we only support one alert being rendered at a time.  (We do that mainly for UX reasons, that it'd look weird to have several visible at once).